geode-redis/src/distributedTest/java/org/apache/geode/redis/PubSubDUnitTest.java

```diff
@@ -46,13 +49,13 @@
   public static void beforeClass() {
     ports = AvailablePortHelper.getRandomAvailableTCPPorts(2);
     Properties redisProps = new Properties();
-    redisProps.put("redis-bind-address", "localhost");
+    redisProps.put(DistributionConfig.REDIS_BIND_ADDRESS_NAME, "localhost");
 
     MemberVM locator = cluster.startLocatorVM(0);
 
-    redisProps.put("redis-port", Integer.toString(ports[0]));
+    redisProps.put(DistributionConfig.REDIS_PORT_NAME, Integer.toString(ports[0]));
```

Comment:
Are these truly internal properties? Maybe there Redis module has it's own set of public properties.